test/api/delete-this.test.ts

```diff
@@ -1,5 +1,3 @@
-describe('delete-this', () => {
-  it('is a dummy test', () => {
-    expect(true).toBeTruthy()
-  })
+test('dummy test', () => {
+  expect(true).toBe(true)
```

Comment:
@maany sorry, deleted this one many commits ago but then noticed part of the CI tests fail without this. I put back a quick (but different) dummy test